GANDLF/models/efficientnet.py

```diff
@@ -395,7 +395,7 @@ def __init__(self, parameters: dict, scale_params):  # how to scale depth and wi
         elif self.n_dimensions == 3:
             self.output_size = (1, 1, 1)
         else:
-            sys.exit("Only 2D or 3D convolutions are supported.")
+            logging.error("Only 2D or 3D convolutions are supported.")
```

Comment:
Same as above, we should still throw error

Reply:
Also when I think of it now, this check appears across many models (as it should). Perhaps it could be checked somewhere globally before params arrive to the model constructors.